manifests/e/Epilogue/EpilogueOperator/v0.8.2-beta/Epilogue.EpilogueOperator.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Epilogue.EpilogueOperator
+PackageVersion: v0.8.2-beta

```

```suggestion
PackageVersion: 0.8.2
```
This must match Control Panel.